packages/remark-package-commands/src/package-execute.ts

```diff
@@ -0,0 +1,64 @@
+import type { Code, Root } from "mdast";
+import type { Transformer } from "unified";
+import { visit } from "unist-util-visit";
+
+import type { RemarkPackageOptions } from "./utils.js";
+import { defaultRegistryPackageManagers, generatePackageManagerTabs, getCodeContent, isValidCommand } from "./utils.js";
+
+// Supported language aliases for the code blocks
+const SUPPORTED_ALIASES = ["package-execute"] as const;
+type SupportedAlias = (typeof SUPPORTED_ALIASES)[number];
+
+/**
+ * Generates multiple tabs of code blocks for different package managers from registry execution commands.
+ * Converts commands like "codama run js" to appropriate registry execution commands:
+ * - npm: npx codama run js
+ * - pnpm: pnpm dlx codama run js
+ * - yarn: yarn dlx codama run js
+ * - bun: bunx codama run js
```

Comment:
For `npx` you probably want to add `@latest` here: `npx codama@latest run js`. Otherwise it'll use outdated dependencies if you run it, then a later version is released with updated dependencies. See https://github.com/npm/cli/issues/4108

I know `pnpx` doesn't have this issue, but I'm unsure about the others. It's probably harmless to add `@latest` in any case though.